config/forms/2021_2022.json

```
@@ -4573,12 +4573,12 @@
                 }
               }
             },
-            "check_income_amount": {
+            "net_income_value_check": {
               "depends_on": [{ "net_income_soft_validation_triggered?": true }],
               "title_text": "Net income is outside the expected range based on the main tenant’s working situation",
```

We’re using ‘lead tenant’ throughout the service:

```suggestion
              "title_text": "Net income is outside the expected range based on the lead tenant’s working situation",
```